main.go

```diff
@@ -118,6 +124,17 @@ var (
 	}
 )
 
+type suggestionNotFoundErr struct {
+	suggestions []string
+}
+
+func (e suggestionNotFoundErr) Error() string {
```

Comment:
the name makes me think this error happens when no suggestions are found, but seems like its not it... maybe another name?

💅: I think prefixing the error with `err` is more common than suffixing with it

Reply:
Hmmm very good point. I had it as `dbNotFoundErr` before which I think makes more sense. Also will change `e` -> `err`

Reply:
Oh wait or did you mean that it's more common to see `errDbNotFound` vs `dbNotFoundErr`?

Reply:
I think so, yes, at least in the stdlibrary... eg. `os.ErrNotExists` and many others... although all of those are public, haven't checked the private ones 🤔

Reply:
Just did a quick check - the private ones also prefix with `err` instead of having it as a suffix. Thanks for the tip! It's a great call to use stdlib for conventions in your own code 💭